UI/installer/mp-installer.nsi

```diff
@@ -446,22 +464,25 @@ Section "un.obs-studio Program Files" UninstallSection1
 	RMDir "$INSTDIR\OBS Studio"
 SectionEnd
 
-Section /o "un.User Settings" UninstallSection2
+Section /o "un.Settings, Scenes, etc" UninstallSection2
 	RMDir /r "$APPDATA\obs-studio"
 SectionEnd
 
+# These two don't work for the uninstaller
+# !define MUI_UNCOMPONENTSPAGE_TEXT_DESCRIPTION_INFO "Choose which aspects of OBS you want to uninstall."
+# !define MUI_UNCONFIRMPAGE_TEXT_TOP "Check the compoonents you want to uninstall. Keeping Settings unchecked is recommended."
```

Comment:
Do we need these commented here?  I'm assuming the current text in the uninstaller is auto-generated?
![image](https://user-images.githubusercontent.com/624931/82203188-f76da900-98d0-11ea-8779-5128ed894365.png)

Reply:
They're commented because they're supposed to work but I couldn't figure out the correct key. Yes, the text you see in your screenshot is auto-generated.

Reply:
I couldn't find documentation to suggest that MUI_UNCOMPONENTSPAGE_TEXT_DESCRIPTION_INFO exists.  MUI_UNCONFIRMPAGE_TEXT_TOP is documented, however.

Reply:
I figured this out.  You need to define `MUI_COMPONENTSPAGE_TEXT` settings _before_ the insertmacro for MUI_UNPAGE_COMPONENTS.
https://nsis.sourceforge.io/Docs/Modern%20UI/Readme.html#toggle_pgc
![image](https://user-images.githubusercontent.com/624931/82221543-23e3ee00-98ee-11ea-86e7-221d587563c0.png)

Reply:
As for MUI_UNPAGE_CONFIRM, you would need to use that with an associated MUI_UNPAGE_CONFIRM insertmacro.
https://nsis.sourceforge.io/Docs/Modern UI/Readme.html#toggle_pgu

Reply:
Thanks for digging into it. I've updated it based on what you found.

Reply:
Hmm, looks like it still doesn't work. What did I do incorrectly?

Reply:
I deleted my old work file for this, but I'm pretty sure you need to define MUI_COMPONENTSPAGE_TEXT_TOP not MUI_UNCOMPONENTSPAGE_TEXT_TOP.  Same with MUI_UNCOMPONENTSPAGE_TEXT_COMPLIST.

Reply:
You would be correct, that fixes it. Pushing the change now.